src/trading/ClaimProceeds.sol

```diff
@@ -57,11 +57,10 @@ contract ClaimProceeds is Controlled, ReentrancyGuard, IClaimProceeds {
     }
 
     function calculateProceeds(IMarket _market, IReportingToken _winningReportingToken, uint8 _outcome, uint256 _numberOfShares) public constant returns (uint256) {
-        uint256 _completeSetCostInAttotokens = _market.getCompleteSetCostInAttotokens();
         uint256 _payoutNumerator = _winningReportingToken.getPayoutNumerator(_outcome);
-        uint256 _getPayoutDenominator = _market.getPayoutDenominator();
+        uint256 _payoutDenominator = _market.getPayoutDenominator();
         // NOTE: rounding error here will result in _very_ tiny amounts of denominationToken left in the market
-        return _numberOfShares.mul(_completeSetCostInAttotokens).div(10**18).mul(_payoutNumerator).div(_getPayoutDenominator);
```

Comment:
Just curious if you know why we were doing `mul(_completeSetCostInAttotokens).div(10**18)` before and why we aren't now. I'm assuming the payoutNumerator includes the total set cost in its calculation then but this should've been wrong before then.

Reply:
This was the bug I referred to in the PR description.  Things were broken before, and this fixes it by changing the way complete set size is calculated.  Now 1 indivisible complete set == 1 indivisible share in each outcome and it costs 1 indivisible unit of currency.  Previously there were some weird calculations involved (like randomly dividing by 10**18) that broke for scalar markets.